micrometer-core/src/main/java/io/micrometer/core/instrument/ObserverHolder.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2017 Pivotal Software, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.micrometer.core.instrument;
+
+/**
+ * @author Dmitry Poluyanov
+ * @since 23.07.17
+ */
+public interface ObserverHolder {
```

Comment:
eyes are crossed.. I can't see any difference between this and Observer. The combination of this + adding Abstracts seems a bit more types than needed at least today. Libraries with a lot of types to say the same thing can put off certain users. Is this pulling its cognitive weight?